api/File.json

```diff
@@ -29,6 +29,20 @@
           "ie": {
             "version_added": "10"
           },
+          "nodejs": [
+            {
+              "version_added": "20.0.0",
+              "notes": "Before 23.0.0, <code>File</code> instance cann't be cloned."
+            },
+            {
+              "alternative_name": "buffer.File",
+              "version_added": "19.2.0",
+              "notes": [
+                "Experimental implementation.",
+                "Must be imported using <code>require('buffer').File</code> or <code>import { File } from 'buffer'</code>."
```

Comment:
I'm not sure about this note, as the alternative name could already imply that this is how `File` can be used.

Reply:
some other cases:

![image](https://github.com/user-attachments/assets/481af103-0aae-4248-803d-938d69575761)

![image](https://github.com/user-attachments/assets/d04aea9c-ce41-430d-b252-38e7dd9e750c)

![image](https://github.com/user-attachments/assets/9d1634b4-41ba-468b-b851-864aff703291)

![image](https://github.com/user-attachments/assets/35165ce3-d775-42c0-9150-281d5eb91ac4)

![image](https://github.com/user-attachments/assets/08bbc122-219d-4c47-8da3-fb3893d4aa1d)

![image](https://github.com/user-attachments/assets/91cce0c1-72df-4913-b382-f05c0b4dafc2)

![image](https://github.com/user-attachments/assets/064d32a4-2527-48d1-bb72-a4546abd6e31)

![image](https://github.com/user-attachments/assets/4368cf76-ac70-440b-870f-42e9c1030907)

![image](https://github.com/user-attachments/assets/20b07d18-42e1-46b8-bc6a-e4ecbca462f2)

Reply:
some opioions:

- *alternative_name* - prefer not to  be kept because it is not the correct format for developers to use it
- *version_removed* - should not be kept if the interface is still available via the specific module even after available via global directly
- *partial_implementation* - should be added if the feature is not available via global directly
- *notes* - need to indicate it is only available via the specific module, but it is not required to indicate how to use the feature via the specific module